### PR TITLE
remove spaces between pdf replicas in gridpack_generation.sh

### PR DESCRIPTION
**Pull request by perrozzi**
otherwise mg_amc crashes...

### CODE REVIEW

**Comment by perrozzi** [comment]
@kdlong please sign off

File: bin/MadGraph5_aMCatNLO/gridpack_generation.sh [hunk 1]
```diff
@@ -478,11 +478,11 @@ if grep -q -e "\$DEFAULT_PDF_SETS" -e "\$DEFAULT_PDF_MEMBERS" $CARDSDIR/${name}_
...
-            sed -i "s/\$DEFAULT_PDF_MEMBERS/True,  False, False, False, False, False, False, False, False, True,  True,False,False,True, False,False,False,True, True, False,True, True, True, True, True, True, True, True, True, True, True, True,  False, False, False, False, False/" ./Cards/run_card.dat 
+            sed -i "s/\$DEFAULT_PDF_MEMBERS/True,False,False,False,False,False,False,False,False,True,True,False,False,True,False,False,False,True,True,False,True,True,True,True,True,True,True,True,True,True,True,True,False,False,False,False,False/" ./Cards/run_card.dat 
         else
             # 4F PDF
                   sed "s/\$DEFAULT_PDF_SETS/320900,11082,13091,13191,13202,23100,23300,23490,23600,23790,25410,25510,25570,25605,25620,25710,25770,25805,25840,92000,320900,320500,260400,262400,263400,292000,292400/" $CARDSDIR/${name}_run_card.dat > ./Cards/run_card.dat
```
Comment (kdlong) [inline bin/MadGraph5_aMCatNLO/gridpack_generation.sh]: There's an additional issue here where 320900 is repeated twice. I've been working on the same change, testing now, will make a separate pull request in 5-10 minutes.
